pom.xml

```diff
@@ -113,6 +114,7 @@
     <curator.version>2.7.1</curator.version>
     <guava.version>30.0-jre</guava.version>
     <netty.version>4.1.86.Final</netty.version>
+    <hive-storage-api.version>2.4.0</hive-storage-api.version>
```

Comment:
hive-storage-api.version should be 2.7.2 in spark 3.2?  add this to spark3.2 profile

Reply:
after talk with code auth .three oom error can be removed, if so we don't need to add this dependency.
 @rarexixi  Could you please help double confirm?